src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -317,6 +318,10 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('web_link.xml');
         }
 
+        if (!$config['reset_services']) {
```

Comment:
Instead of registering and removing the service, I'd rather invert that condition and only register the service if the feature has been enabled.

Reply:
Makes sense :+1: